docs/motivation.md

```diff
@@ -0,0 +1,553 @@
+# The Verona language: Core type system
+
+Verona is a language with a behaviour-oriented concurrency model and a region abstraction for ownership.
```

Comment:
General question: who is the audience for this? And what do we want the effect of reading this document to be on them?

Reply:
Add a section on the audience, expectation of the understanding the reader will get, and purpose of the document.

Reply:
Newspaper style... important first, and less important as you go.

Reply:
Tree of documents versus newspaper style?  To be decided